js/ui/scheduler/workspaces/ui.scheduler.work_space.js

```diff
@@ -271,7 +271,10 @@ var SchedulerWorkSpace = Widget.inherit({
 
         if(isMultiSelection) {
             $cell = this._correctCellForGroup($cell);
-            var $targetCells = this._getCellsBetween($cell, this._$prevCell);
+            let orientation = this.option("type") === "day" && (!this.option("groups").length || this.option("groupOrientation") === "vertical")
```

Comment:
Extract `this.option("type") === "day" && (!this.option("groups").length || this.option("groupOrientation") === "vertical")`  to the separate variable

Reply:
I suppose that is unnecessary because of the single usage of this logical expression. If we will need this expression later, I will extract it ASAP.